src/sentry/quotas/base.py

```diff
@@ -322,6 +324,84 @@ def get_key_quota(self, key):
         limit, window = key.rate_limit
         return _limit_from_settings(limit), window
 
+    def get_project_abuse_quotas(self, org):
+        # Per-project abuse quotas for errors, transactions, attachments, sessions.
+        global_abuse_window = options.get("project-abuse-quota.window")
+
+        for option, compat_options, id, categories in (
+            (
+                "project-abuse-quota.error-limit",
+                (
+                    "sentry:project-error-limit",
+                    "getsentry.rate-limit.project-errors",
+                ),
+                "pae",
+                DataCategory.error_categories(),
+            ),
+            (
+                "project-abuse-quota.transaction-limit",
+                (
+                    "sentry:project-transaction-limit",
+                    "getsentry.rate-limit.project-transactions",
+                ),
+                "pat",
+                (DataCategory.TRANSACTION,),
+            ),
+            (
+                "project-abuse-quota.attachment-limit",
+                (),
+                "paa",
+                (DataCategory.ATTACHMENT,),
+            ),
+            (
+                "project-abuse-quota.session-limit",
+                (),
+                "pas",
+                (DataCategory.SESSION,),
+            ),
+        ):
+            limit = 0
+            abuse_window = global_abuse_window
+            # compat_options were previously present in getsentry
+            # for errors and transactions. The first one is the org
+            # option for overriding the global option, the second one.
+            # For now, these deprecated ones take precedence over the new
+            # to preserve existing behavior.
+            if compat_options:
+                limit = org.get_option(compat_options[0])
+                if not limit:
+                    limit = options.get(compat_options[1])
+
+            if not limit:
+                limit = org.get_option(option)
+                if not limit:
+                    limit = options.get(option)
+
+            if limit == 0:
```

Comment:
This will end up working, but it's a bit brittle:

1. `limit = 0`
2. `org.get_option()` overwrites this with `None`. Note that org options don't have defaults.
3. `options.get()` overwrites this with a default of `0` again. 

If you were to switch the global and org option, this would not work. I think it's safer to use `if not limit` here.